### PR TITLE
Add mobilenet tests add standardised testing module

### PR DESCRIPTION
**Pull request by boba-and-beer**

### CODE REVIEW

File: vectorhub/encoders/image/tfhub/mobilenet.py [hunk 1]
```diff
@@ -55,14 +91,53 @@ def init(self, model_url: str):
         ])
         self.model.build([None, self.image_dimensions, self.image_dimensions, 3])
     
+    def _read(self, image: str):
```
Comment (JackyKoh) [inline vectorhub/encoders/image/tfhub/mobilenet.py:94]: This change feels unnecessary. This means there is now 2 read functions to update, one from base one in mobile-net.

File: tests/test_utils.py [hunk 2]
```diff
@@ -31,3 +31,62 @@ def assert_vector_works(vector, vector_length=None):
...
+            self.assert_bulk_encode_works()
+        elif self.model_type == 'bi_encoder':
+            self.assert_biencode_works()
+            self.assert_bulk_biencode_works()
+    
+    def assert_biencode_works(self):
+        if self.data_type == 'text':
+            assert_vector_works(self.model.encode_question(self.sentence), self.vector_length)
+
+    def assert_bulk_biencode_works(self):
```
Comment (JackyKoh) [inline tests/test_utils.py:76]: think you were meant to have this in the assert_biencode_works

Comment (JackyKoh) [reply tests/test_utils.py:76]: the encode_answer bit

File: vectorhub/encoders/image/tfhub/mobilenet.py (hunk 1 above)
Comment (boba-and-beer) [reply vectorhub/encoders/image/tfhub/mobilenet.py]: Removed

File: tests/test_utils.py (hunk 2 above)
Comment (boba-and-beer) [reply tests/test_utils.py:76]: Added

File: vectorhub/encoders/image/tfhub/mobilenet.py (hunk 1 above)
Comment (boba-and-beer) [reply vectorhub/encoders/image/tfhub/mobilenet.py]: My initial intention was to introduce a bulk_read function to use with the bulk_encode function.

**Pull request merged**